deploy/infra/lib/common-fate-stack-prod.ts

```diff
@@ -171,6 +172,19 @@ export class CommonFateStackProd extends cdk.Stack {
       }
     );
 
+    const sdkApiJwtAudience = new CfnParameter(this, "SDKAPIJWTAudience", {
```

Comment:
I would consider marking these as experimental params, because the SDK is not fully designed yet

```suggestion
    const sdkApiJwtAudience = new CfnParameter(this, "SDKAPIJWTAudience", {
```